### PR TITLE
pontusfagerstrom/cob-148-lock-down-mediasoup-client-and-commit-yarnlock-to-jslib

### PR DESCRIPTION
**Pull request by pnts-se-whereby**
This PR will:
* pin down mediasoup-client to its current version in production.
* remove yarn.lock from gitignore
* commit a newly generated yarn.lock file

As we see Versatica recently have introduce regressions on Mediasoup server-side (in patch releases), we should be a bit cautious on client-side as well.

<details>
  <summary>📦 Published PR as canary version: <code>1.3.2--canary.26.6586818722.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @whereby/jslib-media@1.3.2--canary.26.6586818722.0
  # or 
  yarn add @whereby/jslib-media@1.3.2--canary.26.6586818722.0
  ```
</details>

### CODE REVIEW

**Review by richtrwhereby — APPROVED** [review]
LGTM 🚀

**Comment by pnts-se-whereby** [comment]
@richtrwhereby actually, I think we need to add `--frozen-lockfile` in the release workflow as well.
@havardholvik ?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 node_modules/
-test-report/
-yarn.lock
+test-report/
```
Comment (havardholvik) [inline .gitignore:2]: Add missing newline please 🙏

**Review by havardholvik — COMMENTED** [review]
I guess we also need `--frozen-lockfile` in all our workflows?

**Comment by pnts-se-whereby** [comment]
> I guess we also need `--frozen-lockfile` in all our workflows?

Done.

**Pull request merged**